Content.Server/Singularity/EntitySystems/RadiationCollectorSystem.cs

```diff
@@ -36,7 +56,32 @@ private void OnInteractHand(EntityUid uid, RadiationCollectorComponent component
 
         private void OnRadiation(EntityUid uid, RadiationCollectorComponent component, OnIrradiatedEvent args)
         {
-            if (!component.Enabled) return;
+            if (!component.Enabled || component.RadiationReactiveGases == null)
```

Comment:
Would there be any sense in preserving current behavior (no gas tank needed) if `component.RadiationReactiveGases`, just in case some mapper somewhere wants a radiation collector with the old behavior?

Reply:
IMO, I don't think it's necessary. If mappers want power generators without upkeep, there are others they can use